### PR TITLE
[#9501] Regenerate our Cython

### PR DESCRIPTION
**Pull request by hawkowl**
https://twistedmatrix.com/trac/ticket/9501

### CODE REVIEW

**Review by glyph — APPROVED** [review]
See trac.

File: src/twisted/test/test_shortcut.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 except ImportError:
     skipReason = "Only runs on Windows with win32com"
 
+if sys.version_info[0:2] >= (3, 7):
```
Comment (altendky) [inline src/twisted/test/test_shortcut.py]: Doesn't the full comparison (without `[0:2]`) work fine?  Also, I would have expected we would want to fix the test, not skip it?  Or is there some reason this shouldn't be supported.

Comment (altendky) [inline src/twisted/test/test_shortcut.py:18]: Doesn't the full comparison (without `[0:2]`) work fine?  Also, I would have expected we would want to fix the test, not skip it?  Or is there some reason this shouldn't be supported.

Comment (hawkowl) [reply src/twisted/test/test_shortcut.py:18]: shortcut was only relevant 10 years ago, so it's something we'll get rid of eventually.

**Pull request merged**